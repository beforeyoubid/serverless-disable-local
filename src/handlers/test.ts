export async function handler(event: AWSLambda.APIGatewayProxyEvent, context: AWSLambda.Context) {
  return await Promise.resolve({
    statusCode: 200,
    headers: {
      'Content-Type': 'application/json; charset=utf-8',
    },
    body: JSON.stringify({ message: 'Success!' }),
  });
}
