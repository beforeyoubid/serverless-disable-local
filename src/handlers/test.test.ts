import { handler } from './test';

describe('handler', () => {
  it('should return a response', async () => {
    const result = await handler({} as AWSLambda.APIGatewayProxyEvent, {} as AWSLambda.Context);
    expect(result).toMatchSnapshot();
  });
});
