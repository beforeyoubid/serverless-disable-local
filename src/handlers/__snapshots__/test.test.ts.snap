// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`handler should return a response 1`] = `
{
  "body": "{"message":"Success!"}",
  "headers": {
    "Content-Type": "application/json; charset=utf-8",
  },
  "statusCode": 200,
}
`;
